editor/src/components/canvas/controls/grid-controls.tsx

```diff
@@ -2078,3 +2092,36 @@ function useAllGrids(metadata: ElementInstanceMetadataMap) {
     return MetadataUtils.getAllGrids(metadata)
   }, [metadata])
 }
+
+function useIsSelectedGridItemWithoutInteraction() {
+  const selectedViewsRef = useRefEditorState((store) => store.editor.selectedViews)
+
+  const isItemInteractionActive = useEditorState(
+    Substores.canvas,
+    (store) => {
+      if (store.editor.canvas.interactionSession == null) {
+        return false
+      }
+      return (
+        // movement
+        store.editor.canvas.interactionSession.activeControl.type === 'GRID_CELL_HANDLE' ||
+        // resize (cell)
+        store.editor.canvas.interactionSession.activeControl.type === 'GRID_RESIZE_HANDLE' ||
+        // resize (abs)
+        store.editor.canvas.interactionSession.activeControl.type === 'RESIZE_HANDLE'
+      )
+    },
+    'useShouldHaveVisibleControls isItemInteractionActive',
+  )
+
+  return React.useCallback(
+    (grid: GridData) => {
+      const gridItemSelected = selectedViewsRef.current.some((path) =>
+        EP.isDescendantOf(path, grid.identifier.path),
```

Comment:
This is only true when grid.identifier is a GridContainerIdentifier. Currently on master this works, but after https://github.com/concrete-utopia/utopia/pull/6636 , when a grid item is selected, this is going to be a GridItemIdentifier.

Reply:
I'll wait and update once #6640 is in

Reply:
done